### ISSUE TITLE
[WorldModelGW] Gradient explosion

### ISSUE THREAD

**Issue by jackdawe** [jackdawe/joliRL#16]
While training some unlucky batches can trigger a gradient explosion which then stucks the state loss at a very high local minima.
Reducing the learning rate or the batch size tend to lower the probability for that bug to happen.

**Comment by jackdawe** [comment]
If the problem occurs, training resumes on a previous checkpoint.
